### PR TITLE
Generate Invoice Page Mobile : section 2 

### PR DESCRIPTION
**Pull request by Shruti-Apte**
### **Notion :**
Section 2
https://www.notion.so/saeloun/Mobile-Responsive-Generate-Invoice-Page-Invoice-Details-section-359a7ae203be40a1b8123bf8cedb37ec?pvs=4 
### **What :**
 - Line Items section of generate invoice page mobile view.
 - User can add line entry.
 - User can edit line entry.
 - User can delete line entry.
 - User can add multiple line entries. 
 - Added static footer (will integrate values in 3rd section)
### **Why :**
- Requirement & Enhancement 
### **Preview :**
**Screenshots:**
Screenshot 2023-03-22 at 6 18 06 PM

Screenshot 2023-03-22 at 6 20 57 PM

Screenshot 2023-03-21 at 5 51 38 PM

Screenshot 2023-03-23 at 4 17 48 PM

Screenshot 2023-03-21 at 5 51 55 PM

**Video:**
https://www.loom.com/share/bb1b8d98c4d842b08749ab1b4aff1ade
### **Todos** (for testing):
- Tested all basic workflows.

### CODE REVIEW

File: app/javascript/src/common/FormikFields/InputErrors/index.tsx [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
-  <div className="mx-0 mb-6 block text-xs tracking-wider text-red-600">
-    {fieldErrors && fieldTouched && <div>{fieldErrors}</div>}
-  </div>
-);
+const InputErrors = ({ fieldErrors, fieldTouched }) =>
+  fieldErrors && fieldTouched ? (
+    <div className="mx-0 mb-6 block text-xs tracking-wider text-red-600">
+      <div>{fieldErrors}</div>
+    </div>
+  ) : null;
```
Comment (prasanthchaduvula) [inline app/javascript/src/common/FormikFields/InputErrors/index.tsx:8]: Why do we want to return `null`

File: app/javascript/src/components/Invoices/Generate/MobileView/Container/MenuContainer/InvoiceDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+                )}
+              </div>
+            </div>
+            <div className="flex justify-between py-3">
+              <InputField
+                id="invoiceNumber"
+                inputBoxClassName="border focus:border-miru-han-purple-1000"
+                label="Invoice Number"
+                name="invoiceNumber"
+                readOnly={false}
```
Comment (prasanthchaduvula) [inline app/javascript/src/components/Invoices/Generate/MobileView/Container/MenuContainer/InvoiceDetails.tsx:239]: You already `readOnly={false}` default value right

**Review by prasanthchaduvula — CHANGES_REQUESTED** [review]

**Comment by prasanthchaduvula** [comment]
@Shruti-Apte Record video and share it here - Please add more description to timesheet entries and add them in lint items

**Comment by Shruti-Apte** [comment]
Closing this PR as section 3  PR (1193) has all the changes from this PR.

**Pull request closed without merging**